src/widgets/splits/Split.hpp

```diff
@@ -118,6 +120,7 @@ class Split : public BaseWidget, pajlada::Signals::SignalHolder
     pajlada::Signals::Connection roomModeChangedConnection_;
 
     pajlada::Signals::Connection indirectChannelChangedConnection_;
+    pajlada::Signals::SignalHolder SignalHolder_;
```

Comment:
should be `signalHolder_` not `SignalHolder_`

Reply:
Fixed in 7ee5716b

Reply:
Fixed in 7ee5716b